library/core/registry/src/main/java/org/quiltmc/qsl/registry/impl/RegistryConfig.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2022 QuiltMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.quiltmc.qsl.registry.impl;
+
+import org.quiltmc.config.api.Config;
+import org.quiltmc.config.api.values.TrackedValue;
+import org.quiltmc.loader.api.config.QuiltConfig;
+
+public class RegistryConfig {
```

Comment:
```suggestion
@ApiStatus.Internal
public class RegistryConfig {
```

I also would appreciate more blank lines in this class, everything is so close together right now.